OpenRA.Mods.RA/SupportPowers/ParatroopersPower.cs

```diff
@@ -36,19 +49,6 @@ public class ParatroopersPowerInfo : SupportPowerInfo
 		[Desc("Troops to be delivered.  They will be distributed between the planes if SquadSize > 1.")]
 		public string[] DropItems = { };
```

Comment:
The real problem is that this field isn't marked `readonly`.  Please add this and then move the other fields back to where they were below this.